package.json

```
@@ -29,20 +29,21 @@
     "build:confluence": "npm run clean && npm run css:confluence && npm run fonts && npm run images && npm run js:confluence"
   },
   "devDependencies": {
-    "autoprefixer": "7.2.6",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-es2015": "6.24.1",
-    "babelify": "7.3.0",
-    "browserify": "16.5.1",
-    "eslint": "6.8.0",
-    "fs-extra": "3.0.1",
-    "glob": "7.1.3",
-    "handlebars": "4.7.6",
-    "jquery": "3.5.0",
-    "postcss-cli": "7.1.0",
-    "sass": "1.32.13",
-    "uglify-js": "3.4.9",
-    "zip-a-folder": "0.0.6"
+    "@babel/core": "^7.14.3",
+    "@babel/preset-env": "^7.14.4",
+    "autoprefixer": "10.2.6",
+    "babelify": "10.0.0",
+    "browserify": "17.0.0",
+    "eslint": "7.27.0",
+    "fs-extra": "10.0.0",
+    "glob": "7.1.7",
+    "handlebars": "4.7.7",
+    "jquery": "3.6.0",
+    "postcss": "^8.3.0",
+    "postcss-cli": "8.3.1",
+    "sass": "1.34.0",
+    "uglify-js": "3.13.8",
+    "zip-a-folder": "1.0.1"
```

```suggestion
    "@babel/core": "7.14.3",
    "@babel/preset-env": "7.14.4",
    "autoprefixer": "10.2.6",
    "babelify": "10.0.0",
    "browserify": "17.0.0",
    "eslint": "7.27.0",
    "fs-extra": "10.0.0",
    "glob": "7.1.7",
    "handlebars": "4.7.7",
    "jquery": "3.6.0",
    "postcss": "8.3.0",
    "postcss-cli": "8.3.1",
    "sass": "1.34.0",
    "uglify-js": "3.13.8",
    "zip-a-folder": "1.0.1"
```

Renovate will complain about the range of versions. :(